src/fluree/db/query/exec/where.cljc

```diff
@@ -29,6 +29,30 @@
                                 {:status 400 :error :db/invalid-query})))
     :else     :spot))
 
+(defn augment-object-fn
+  "Returns a pair consisting of an object value and boolean function that will
+  return false when applied to object values whose flake should be filtered out
+  of query results. This function augments the original object function supplied
+  in an object pattern under the `::fn` key (if any) by also checking if a
+  prospective flake object is equal to the supplied `o` value if and only if the
+  `:spot` index is used, the `p` value is `nil`, and the `s` and `o` values are
+  not `nil`b. In this case, the new object value returned by this function will
```

Comment:
Little typo in the second sentence of the docstring:
" not `nil`b. " has an extraneous "b" at the end